components/servo/lib.rs

```diff
@@ -236,7 +236,7 @@ where
     pub fn new(
         opts: Opts,
         preferences: Preferences,
-        rendering_context: RenderingContext,
+        rendering_context: Rc<dyn RenderingContext>,
```

Comment:
Can this be a generic instead of trait object? I don't think we will hot reload the context after creation.